onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -40,13 +41,84 @@ std::string GetEnginePath(const ::std::string& root, const std::string& name) {
   }
 }
 
-std::string GetVecHash(const ::std::vector<int>& vec) {
+std::string GetProfilePath(const ::std::string& root, const std::string& name) {
```

Comment:
we could have made GetEnginePath generic (i.e. support any file extension) rather than duplicate the code.